app/controllers/items_controller.rb

```diff
@@ -1,6 +1,10 @@
 class ItemsController < ApplicationController
   def index
-    @items = Item.all
+    if !params[:search].nil?
+      @items = Item.search(params[:search]).order("created_at DESC")
```

Comment:
minor syntax thing. rails 4 allows you to do `.order(created_at: :desc)`, which is a little neater.